### ISSUE TITLE
Requêtes cross-origin

### ISSUE THREAD

**Issue by bchartier** [geobretagne/mviewer#190]
Il me semble que le sujet des requêtes cross-origin et de l'usage des proxy revient souvent (exemple #161). N'aurions-nous pas intérêt à faire un chapitre spécifique dans la doc de mviewer ?

**Comment by spelhate** [comment]
Je suis d'accord.

**Comment by fxduponcheel** [comment]
Bonjour Benjamin et Stéphane

Effectivement ça serait bien utile !

Depuis ma migration en https (merci Benjamin), j'ai de nouveau un pbm de cross-origin avec les flux de http://administration.carmencarto.fr/services/catalogue/119

message de la console : 
Blocage du chargement du contenu mixte actif (mixed active content) « http://ws.carmencarto.fr/WMS/119/fxx_inpn?SERVICE=WMS&VERSIO…2C6057175.869430507%2C457643.77574900724%2C6368427.448607745 »

Il n'y a pas de protocole https proposé par carmencarto. On est obligé de mixer https et http.

accès directe à la donnée posant pbm : 
https://geocouesnon.sage-couesnon.fr/geocouesnon/?x=-12010&y=6212702&z=9&l=Sites_Ramsar*%252Ccours_eaudbl1*%252Cbassin_versant_couesnon*&lb=osm&mode=d

**Comment by spelhate** [comment]
Le passage à https impique que les ressources chargées proviennent également de https. Pour Carmen, https n'est pas disponible.
Sur GéoBretagne, nous utilisons le paramètre de layer **useproxy="true"** pour lever cette contrainte. A noter que ce paramétrage fonctionne seulement si mviewer dispose d'un proxy Ajax. C'est le cas sur GéoBretagne.

**Comment by fxduponcheel** [comment]
Est ce compliqué d'installer un proxy ajax ? Ca s'installe où ? sur mon Geoserver ? sur le mviewer ?

**Comment by spelhate** [comment]
C'est une brique supplémentaire à mviewer. Sur GéoBretagne, nous utilisons le proxy de GeOrchestra.
Mais il y a d'autres possibilités. Il existe des proxys ajax en php, en python en java... Par contre je n'ai pas connaissance d'une installation simple.

**Comment by lecault** [comment]
Est-ce encore d'actualité avec la normalisation du passage https ?

**Comment by vfabry** [comment]
En Hauts-de-France, tous les services web produits par la DREAL sont toujours en http... :-(

**Comment by Gaetanbrl** [comment]
Toute contrib dans la doc est la bienvenu si une section peut aider sur le proxy. Une note sur l'utilisation fortement conseillée du HTTPS serait dans ce cas la bienvenu.
 Mais pour moi ca doit plus être une référence anecdotique sur ce qu'il est possible de faire plus qu'un tuto pas à pas.
